rails/benchmarks/bm_activerecord_validations_valid.rb

```diff
@@ -1,46 +1,6 @@
 require_relative 'support/activerecord_validations_base.rb'
 require_relative 'support/benchmark_rails.rb'
 
-require 'bundler/setup'
-require 'rails'
-require 'active_record'
-
-require_relative 'support/benchmark_rails.rb'
-
-class Post < ActiveRecord::Base; end
-
-ActiveRecord::Base.establish_connection(
-  adapter: 'sqlite3',
-  database: ':memory:'
-)
-
-ActiveRecord::Migration.verbose = false
-
-ActiveRecord::Schema.define do
-  create_table :posts, force: true do |t|
-    t.string :title
-    t.string :author
-    t.text :body
-    t.integer :sequence
-    t.integer :age
-    t.string :subdomain
-    t.string :legacy_code
-    t.string :size
-  end
-end
-
-class Post < ActiveRecord::Base
-  validates :title, presence: true, confirmation: true
-  validates :sequence, uniqueness: true
-  validates :age, numericality: { greater_than: 18, less_than: 80 }
-  validates :subdomain, exclusion: { in: %w(www us ca jp),
-    message: "%{value} is reserved." }
-  validates :legacy_code, format: { with: /\A[a-zA-Z]+\z/,
-    message: "only allows letters" }
-  validates :size, inclusion: { in: %w(small medium large),
-    message: "%{value} is not a valid size" }
-end
-
```

Comment:
Yup I think we should intentionally duplicate code in this case.

![screenshot from 2015-04-19 08 53 49](https://cloud.githubusercontent.com/assets/4335742/7217700/a50ce4c6-e671-11e4-9974-e1dcdc18f000.png)

The above makes it easier to read from the site what we're benchmarking. On top of that, the benchmarks are not meant to change often so it'll be fine if we duplicate code here.